src/app/productManagement/pricing/templates/updateProductPrice.modal.html

```diff
@@ -1,4 +1,4 @@
-<form name="updateProductPriceModal.form">
+<form name="updateProductPriceModal.form" ng-submit="updateProductPriceModal.submit()" oc-pretty-submit>
```

Comment:
It doesn't break the price break directives, but if all the user does is delete a price break, the Save button is still disabled. I'll come up with a resolution for that.